airflow/providers/google/marketing_platform/example_dags/example_analytics.py

```diff
@@ -20,12 +20,15 @@
 
 from airflow import models
 from airflow.providers.google.marketing_platform.operators.analytics import (
-    GoogleAnalyticsListAccountsOperator,
+    GoogleAnalyticsListAccountsOperator, GoogleAnalyticsRetrieveAdsLinksListOperator,
 )
 from airflow.utils import dates
 
 default_args = {"start_date": dates.days_ago(1)}
 
+account_id = "12345"
+web_property_id = "web_property_id"
```

Comment:
```
ACCOUNT_ID = os.environ.get("GA_ACCOUNT_ID", "123456789")
WEB_PROPERTY = os.environ.get("GA_WEB_PROPERTY", "UA-12345678-1")
```
This will work with env variables from config file :)

Reply:
Done.